### ISSUE TITLE
Getting error when running make bootstrap

### ISSUE THREAD

**Issue by Verticaleap** [mongodb/vault-plugin-secrets-mongodbatlas#19]
Following the README, when I run the `make bootstrap`, I am getting the following error:

```
Installing/Updating github.com/mitchellh/gox
go: github.com/mongodb/go-client-mongodb-atlas@v0.0.3 requires
        github.com/golangci/golangci-lint@v1.17.1 requires
        github.com/go-critic/go-critic@v0.0.0-20181204210945-1df300866540: invalid pseudo-version: does not match version-control timestamp (2019-05-26T07:48:19Z)
make: *** [bootstrap] Error 1
```

**Comment by Verticaleap** [comment]
This error was found when trying to make with go1.13.1, make bootstrap did work with go1.11.13.

**Comment by thetonymaster** [comment]
Sorry for the last response, I ran this on my own PC and didn't got any error, let me try on a clean environment.

**Comment by themantissa** [comment]
@Verticaleap  wanted to let you know this work is now included in the 1.4Beta1 version of Vault released by HashiCorp and will be in the GA 1.4 Vault version.  I'll close this PR as you won't need to build this on your own now.
